### PR TITLE
Error boundary

### PR DESCRIPTION
**Pull request by andrepimenta**
**Description**

Error boundary for catching errors, logging them to sentry and showing a nice error screen to the user.

### CODE REVIEW

File: app/components/Views/ErrorBoundary/index.js [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+
+	cancelExportSeedphrase = () => {
+		this.setState({ backupSeedphrase: false });
+	};
+
+	getErrorMessage = () => `View: ${this.props.view}\n${this.state.error.toString()}`;
+
+	copyErrorToClipboard = async () => {
+		await Clipboard.setString(this.getErrorMessage());
+		Alert.alert('Copied to clipboard', '', [{ text: 'OK' }], {
```
Comment (estebanmino) [inline app/components/Views/ErrorBoundary/index.js:175]: small thing here, can we move it to strings?

**Review by estebanmino — APPROVED** [review]
LGTM

File: app/components/Views/ErrorBoundary/index.js (hunk 1 above)
Comment (andrepimenta) [reply app/components/Views/ErrorBoundary/index.js:175]: Yes thanks!

**Review by ibrahimtaveras00 — APPROVED** [review]
QA Passed 👍

**Pull request merged**